packages/ozone/src/mod-service/index.ts

```diff
@@ -162,6 +168,25 @@ export class ModerationService {
       })
     }
 
+    // If multiple labels are passed, then only retrieve events where all those labels exist
+    if (addedLabels.length) {
+      builder = builder.where(
+        'createLabelVals',
+        'ilike',
+        `%${addedLabels.join('%')}%`,
+      )
+    }
```

Comment:
I believe this condition would require that the labels appear in a particular order in `createLabelVals`.

Reply:
great catch!